.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy
```

Comment:
Have you tried the whole workflow yet? Just curious.

Reply:
So I don't think I really can until its merged. They have a workflow validator, but no way to test what it'll run.